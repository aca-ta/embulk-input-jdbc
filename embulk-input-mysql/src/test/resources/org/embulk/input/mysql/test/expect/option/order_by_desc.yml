table: test1
order_by: 'id desc'
