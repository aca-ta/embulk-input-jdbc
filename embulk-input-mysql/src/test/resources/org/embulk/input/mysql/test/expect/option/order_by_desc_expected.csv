4,D
3,C
2,B
1,A
